poms/pom.xml

```diff
@@ -64,8 +64,8 @@
     <ohdr.version>1.0.31</ohdr.version>
     <esh.version>0.10.0-SNAPSHOT</esh.version>
     <repo.version>2.4.x</repo.version>
-    <karaf.version>4.1.5</karaf.version>
-    <jetty.version>9.3.21.v20170918</jetty.version>
+    <karaf.version>4.2.1</karaf.version>
+    <jetty.version>9.4.10.v20180503</jetty.version>
```

Comment:
Your test distro actually shows that Karaf bundles version 9.4.11:
```
70 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.client
 71 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.continuation
 72 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.deploy
 73 │ Active   │  80 │ 9.4.10.v20180503        │ org.eclipse.jetty.http
 74 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.http
 75 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.io
 76 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.jaas
 77 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.jmx
 78 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.jndi
 79 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.plus
 80 │ Active   │  80 │ 9.4.10.v20180503        │ org.eclipse.jetty.proxy
 81 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.rewrite
 82 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.security
 83 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.security.jaspi
 84 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.server
 85 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.servlet
 86 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.servlets
 87 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.util
 88 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.util.ajax
 89 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.webapp
 90 │ Active   │  80 │ 9.4.10.v20180503        │ org.eclipse.jetty.websocket.api
 91 │ Active   │  80 │ 9.4.10.v20180503        │ org.eclipse.jetty.websocket.client
 92 │ Active   │  80 │ 9.4.10.v20180503        │ org.eclipse.jetty.websocket.common
 93 │ Active   │  30 │ 9.4.11.v20180605        │ org.eclipse.jetty.xml
```

Reply:
Thanks for spotting that! I got the version from the [dependency upgrade list in the release notes](https://github.com/apache/karaf/blob/master/RELEASE-NOTES.md#dependency-upgrade) but didn't expect to find two issues about upgrading Jetty. :wink: